Dmitriy_Vysotskyy2.0/PageObjects/HomePage.cs

```diff
@@ -0,0 +1,90 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
```

Comment:
Please review all classes and remove using which are not used. You must keep your code clean